community/kernel/src/main/java/org/neo4j/kernel/impl/store/DumpStore.java

```diff
@@ -95,34 +106,37 @@ else if ( !file.isDirectory() && file.getName().indexOf( ':' ) != -1 )
                 throw new IllegalArgumentException( "No such file: " + arg );
             }
         }
-        switch ( file.getName() )
+        try ( NeoStores neoStores = createStoreFactory.apply( file ).openNeoStores( false ) )
```

Comment:
NeoStores opens the stores lazily as they are requested, so it looks like we open all of them, but it's not actually the case.